sample-linux.ini

```diff
@@ -10,6 +10,10 @@ maxCacheEntries = 2000000
 mruSymbolStateFile = /tmp/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
+[DiskCache]
+cachePath = /tmp/snappy/cache
+cacheQuota = 1073741824
```

Comment:
a gig seems like a lot. how many parsed & pickled xul.sym files is that? if it's > 10,000 files it might hurt disk lookup time?
in any case, make sure to test how well the server performs at the quota limit